test/api-test.js

```diff
@@ -190,7 +190,9 @@ test.serial('#makeApiRequest returns rejected promise for 4xx status codes', asy
 		statusText: 'Not found'
 	});
 
-	await t.throwsAsync(() => service.makeApiRequest('notifications'), 'client error');
+	await t.throwsAsync(() => service.makeApiRequest('notifications'), {
```

Comment:
Remove the `() => ` part.

Reply:
How can `t.throwsAsync` catch and compare the error if the call throws before its result is passed to `t.throwsAsync`? 🤔

Reply:
It doesn't throw. `makeApiRequest` returns a promise. You pass a promise to `throwsAsync` and it checks whether it rejects or not.

Reply:
Promises, how do they work